src/index.html

```diff
@@ -231,13 +231,25 @@ <h1>Easy Steps To Finding Your Perfect Roommate</h1>
 				app-logo {
 					margin-top: 100px;
 				}
+                button {
+                    background-color: var(--primary-background);
+                    color: white; 
+                    text-align: center;
+                    padding: 18px 25px;
+                    border-radius: 5px;
+                    font-weight: 500;
+                    margin-top: 50px;
+                }
 			</style>
 			<div class="container animated bounce">
 				<h1>Congratulations!!</h1>
 				<p>We have received your invitation request to be a member of slash the rent.</p>
 				<p>Please be patient while we process your request. You will receive an invitation response via email/text.</p>
 				<app-logo color="white"></app-logo>
 				<!-- <button>Share with a friend</button> -->
+                <section>
```

Comment:
There is no need for the section element since there isn't any major amount of content other than the button. You can remove the section element